Makefile

```diff
@@ -16,85 +69,138 @@ endif
 # Setting SHELL to bash allows bash commands to be executed by recipes.
 # This is a requirement for 'setup-envtest.sh' in the test target.
 # Options are set to exit when a recipe line exits non-zero or a piped command fails.
-SHELL = /usr/bin/env bash -o pipefail
+# SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
-# Allow overriding release-manifest generation destination directory
-RELEASE_DIR ?= out
-
 # Quiet Ginkgo for now.
 # The warnings are in regards to a future release.
 export ACK_GINKGO_DEPRECATIONS := 1.16.5
 export ACK_GINKGO_RC=true
 
-export PROJECT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
-export PATH := $(PROJECT_DIR)/bin:$(PATH)
+export PATH := $(TOOLS_BIN_DIR):$(PATH)
 
 all: build
 
-##@ General
+## --------------------------------------
+## Binaries
+## --------------------------------------
 
-# The help target prints out all targets with their descriptions organized
-# beneath their categories. The categories are represented by '##@' and the
-# target descriptions by '##'. The awk commands is responsible for reading the
-# entire set of makefiles included in this invocation, looking for lines of the
-# file as xyz: ## something, and then pretty-format the target and help. Then,
-# if there's a line with ##@ something, that gets pretty-printed as a category.
-# More info on the usage of ANSI control characters for terminal formatting:
-# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
-# More info on the awk command:
-# http://linuxcommand.org/lc3_adv_awk.php
+.PHONY: binaries
+binaries: $(CONTROLLER_GEN KUSTOMIZE) $(GOLANGCI_LINT) $(STATIC_CHECK) $(GINKGO_V1) $(GINKGO_V2) $(MOCKGEN) $(KUSTOMIZE) managers # Builds and installs all binaries
 
-help: ## Display this help.
-	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
+.PHONY: managers
+managers:
+	$(MAKE) manager-cloudstack-infrastructure
 
- MANIFEST_GEN_INPUTS=$(shell find ./api ./controllers -type f -name "*test*" -prune -o -name "*zz_generated*" -prune -o -print)
```

Comment:
The original makefile from kubebuilder was really weak. It actually took a lot of work to make it build everything based on changed files.

I'm not at all for removing these features from the make file without something to replace them.